README.md

```diff
@@ -39,7 +39,7 @@ Or add `bootstrap-loader` as an entry point in your webpack config:
 entry: [ 'bootstrap-loader', './app' ]
 ```
 
-Config is optional. It can be placed in root dir with name `.bootstraprc`. You can write it in `YAML` or `JSON` formats. Take a look at the default config files for [Bootstrap 3](.bootstraprc-3-default) and [Bootstrap 4](.bootstraprc-4-default). Note, we recommend using a configuration or else you might pick up unwanted upgrades, such as when we make Bootstrap 4 the default.
+Config is optional. It can be placed in root dir with name `.bootstraprc`. You can write it in `YAML` or `JSON` formats. Take a look at the default config files for [Bootstrap 3](.bootstraprc-3-default) and [Bootstrap 4](.bootstraprc-4-default). Note, we recommend using a configuration or else you might pick up unwanted upgrades, such as when we make Bootstrap 4 the default. Also note that config options don't fall back on the defaults once a config file is present. It might be easiest way to copy over a default config file as a starting point.
 
```

Comment:
> It might be easiest way to copy over a default config file as a starting point.

Just this is enough probably, with some bold:

Note, **we recommend using a configuration file** or else you might pick up unwanted upgrades, such as when we make Bootstrap 4 the default. Also note that config options don't fall back on the defaults once a config file is present.